<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nextWindowThird</class>
 <widget class="QMainWindow" name="nextWindowThird">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1525</width>
    <height>1405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nextWindowThird</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>вернуться в главное меню</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>451</width>
      <height>741</height>
     </rect>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>сегодня</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание заметки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>550</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить траты</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить заметку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Прибыль</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>100</y>
      <width>561</width>
      <height>471</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Траты</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>480</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить прибыль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Заголовок заметки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести график</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>720</y>
      <width>224</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>Записать информацию в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>Чтение информации из файла</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 24pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>160</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Установить название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1525</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
